yt_dlp/extractor/khanacademy.py

```diff
@@ -14,97 +17,118 @@ class KhanAcademyBaseIE(InfoExtractor):
     def _parse_video(self, video):
         return {
             '_type': 'url_transparent',
-            'url': video['youtubeId'],
-            'id': video.get('slug'),
-            'title': video.get('title'),
-            'thumbnail': video.get('imageUrl') or video.get('thumbnailUrl'),
-            'duration': int_or_none(video.get('duration')),
-            'description': video.get('description'),
+            'url': video["youtubeId"],
             'ie_key': 'Youtube',
+            **traverse_obj(video, {
+                'id': ('id', {str_or_none}),
```

Comment:
```suggestion
            'url': video['youtubeId'],
            'id': video['youtubeId'],
            'ie_key': 'Youtube',
            **traverse_obj(video, {
                'display_id': ('id', {str_or_none}),
```

`id` should be same as inner extractor (youtube). It's mistake in old IE